client/my-sites/patterns/home/index.tsx

```diff
@@ -34,8 +54,54 @@ export const PatternsHomePage = ( {
 	const locale = useLocale();
 	const isLoggedIn = useSelector( isUserLoggedIn );
 
+	const [ searchTerm, setSearchTerm ] = useState( () => {
+		if ( typeof window !== 'undefined' ) {
+			const params = new URLSearchParams( window.location.search );
+			return params.get( 's' ) ?? '';
+		}
+
+		return '';
+	} );
+
+	const isPatternsDisplaying = Boolean( category || searchTerm );
```

Comment:
The mix of singular verb and plural noun is strange, what do think of the following instead?

```suggestion
	const showListOfPatterns = Boolean( category || searchTerm );
```